### DISCUSSION TITLE
The star counts for the starstruck achievement is very wrong

### DISCUSSION THREAD

**Discussion by gnif** [community/community#21050] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/351151/180356030-089a5497-9dd6-482a-a207-382f506b9379.png)

LookingGlass: ![image](https://user-images.githubusercontent.com/351151/180356038-45e44af8-8676-49c4-b9db-c4645ec0207c.png)

Also mod_rpaf is repeated twice?

**Comment by radar-blip** [comment, marked as answer]
Only the correct amount of stars for each tier (level) will be shown here. 
For more info, see here: https://github.com/drknzz/GitHub-Achievements

**Reply by gnif** [reply]
Ah, I see, however it still doesn't explain why mod_rpaf is listed twice with different counts.

**Reply by radar-blip** [reply]
It doesn't matter the repo, only the star count.

**Reply by TuanLove1** [reply]
thanks

**Comment by initialshoot** [comment]
ok

**Reply by TuanLove1** [reply]
thanks

**Comment by TuanLove1** [comment]
thanks

**Comment by YarinsHyper** [comment]
Star this https://github.com/YarinsHyper/portfolio and ill star yours 👍🏼
